Simplenote/EmailVerification.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+
+
+// MARK: - EmailVerification
+//
+struct EmailVerification {
+    let token: String
+    let status: String
+}
```

Comment:
`token` is optional and won't be there in case account is not yet verified.
Maybe we can make status an `enum`? 
For example:
```
enum EmailVerificationStatus {
    case sent
    case verified
    case other(String) // not sure if we want this one to catch possible unsupported statuses
}
```

Reply:
Updated (with few details discussed over Slack). Thanks for helping me improve this one!!